docs/medical-api/more-info/medical-record-summary.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "Medical Record Summary"
+description: "Consolidate Medical Records Into a Single Document"
+---
+
+This guide will teach you how to generate a Medical Record Summary, which condenses all of patients' medical records into a single document.
+
+To do this, Metriport extracts FHIR resources across all available CDA documents for a patient and de-duplicates them, resulting in a set of unique medical records, which are then written into HTML and PDF files, making them extremely easy to navigate.
+
+Here's the title page of an example Medical Record Summary:
+
+<img className="h-100" src="/images/example-medical-record.png" />
+
+## Generate using API
+
+To generate a Medical Record Summary using Metriport API, you need to follow these steps:
+
+1. Start a [Consolidated Data Query](/medical-api/api-reference/fhir/consolidated-data-query-post). This endpoint allows you to choose [FHIR resources](/medical-api/fhir/overview), date range, and rendering (`PDF` or `HTML`) for your Medical Record Summary. When this query completes, a [webhook](/medical-api/more-info/webhooks) with a JSON payload containing all these resources will be sent to you.
+
+<Tip>
+  Once the query is kicked off, you can monitor its status using the [Get Consolidated Data Query
+  Status](/medical-api/api-reference/fhir/consolidated-data-query-get) endpoint.
+</Tip>
+
+2. In addition to webhooks, you can download the Medical Record Summary in a PDF or HTML format using the [Get Medical Record Summary](/medical-api/api-reference/fhir/get-medical-record-summary) endpoint.
+
+<Tip>
+  Prior to creating or downloading a new Medical Record Summary, you can check when it was created,
+  and whether it's available in either PDF or HTML format using the [Get Medical Record Summary
+  Status](/medical-api/api-reference/fhir/get-medical-record-summary-status) endpoint.
+</Tip>
+
+## Generate using the Dashboard
+
+To generate a Medical Record Summary from the dashboard, you need to navigate to the patient's document section as follows:
+
+<img className="h-100" src="/images/dashboard-patient.png" />
+
+Then, select whether you want to download a new or a previously-generated (if such is available) Medical Record Summary, and click the button with the desired format to initiate the download.
```

Comment:
let's add a warning here that this process may take some time, and to ensure they have pop-ups enabled on their browser